submissions/HannaSyn/frogger-game/app.js

```diff
@@ -0,0 +1,176 @@
+const cellHeight = 83;
+const cellWidth = 101;
+const totalRow = 6;
+const totalCol = 5;
+
+class Position {
+    minY;
+    minX;
+    maxY;
+    maxX;
+    initX;
+    initY;
+    x;
+    y;
+
+    constructor( row, col ) {
+        this.initX = (cellWidth * col) - (cellWidth);
+        this.initY = (cellHeight * row) - (cellHeight / 2);
+        this.x = this.initX;
+        this.y = this.initY;
+        this.maxY = cellHeight * (totalRow - 1);
+        this.maxX = cellWidth * (totalCol - 1);
+        this.minY = cellHeight * (row - 1) - this.initY;
+        this.minX = 0;
+    }
+
+    reset() {
+        this.x = this.initX;
+        this.y = this.initY;
+    }
+
```

Comment:
Please check and remove redundant empty lines (here and later)

Reply:
fixed